### PR TITLE
Layout update user

### PR DESCRIPTION
**Pull request by mvn-nale-dn**
## Description
Create layout update users
## Reference Requirements
- Task ID: #11

## Reference Isuees
#{issue_id}

## Migrations
NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

- Refer to [Using Explain with MySQL](https://viblo.asia/p/su-dung-explain-de-toi-uu-cau-lenh-mysql-BYjv44gmvxpV)

### CODE REVIEW

File: routes/web.php [hunk 1]
```diff
@@ -14,11 +14,9 @@
     return view('welcome');
 
 });
-Route::group(['namespace' => 'Admin', 'prefix' => 'admin' ], function(){
-   Route::resource('users', 'UserController');
-});
 
 Route::group(['namespace' => 'Admin', 'prefix' => 'admin' ], function(){
+	 Route::resource('users', 'UserController');
     Route::get('/', function () {
```
Comment (at-haphan) [inline routes/web.php]: Thêm 1 tab vào

File: resources/views/admin/users/update.blade.php [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+                </div>
+            </form>
+          </div>
+    </section>
+
+    <!-- Main content -->
+    
+    <!-- /.content -->
+</div>
+@endsection
```
Comment (at-haphan) [inline resources/views/admin/users/update.blade.php]: thiếu endline

**Comment by at-haphan** [comment]
LGTM

File: resources/views/admin/users/update.blade.php [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+        <div class="box box-primary">
+            <div class="box-header with-border">
+              <h3 class="box-title">{{trans('user.update')}}</h3>
+            </div>
+            <!-- /.box-header -->
+            <!-- form start -->
+            <form role="form">
+                <div class="box-body">
+                    <div class="form-group">
+                        <label>{{trans('user.form.title-inputs.email')}}</label>
```
Comment (at-huydq) [inline resources/views/admin/users/update.blade.php]: chuyển `title-inputs` sang `title_inputs` giúp a nhé

**Comment by at-huydq** [comment]
LGTM.

**Pull request merged**